experimental/examples/bubblebench/legacy-shared-tree/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist
```

Comment:
all these are in the top level git ignore already an't they? Can we delete these files?

Reply:
Oops, I was wrong about how that works.